### PR TITLE
better explanation for the validators

### PR DESCRIPTION
**Pull request by pb82**
Not really sure what else to add here.

### CODE REVIEW

File: ui/components/common/Validators.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ const Validators = {
  *  ...
  *  <Validation Function N>, <Validation Subject N>
  * ]
+ *
+ * This can be used to perform a number of validations against the available
+ * validators. Used to check dialog input values for correctness.
+ *
```
Comment (camilamacedo86) [inline ui/components/common/Validators.js]: Why is a par length true? What and why need to be check done when the length is an odd number?

Comment (pb82) [reply ui/components/common/Validators.js]: It's always one validator per input value. So the total length of the array % 2 needs to be zero.

Comment (pb82) [reply ui/components/common/Validators.js]: So if you want to validate 4 input values you would have 4 validator functions + 4 input values. Array length needs to be 8.

Comment (josemigallas) [reply ui/components/common/Validators.js]: I find this function confusing, why would you even pass the names of the forms to validate if in the end you just want the reduced validation result?
Also, if validations is a 10 length array and the first one fails, it still iterate over the rest.
Finally, if the input is wrong (say a length 3 array) it returns a false negative.

Comment (josemigallas) [reply ui/components/common/Validators.js]: Sorry I know it's late to say this but I didn't know this even existed.

Comment (pb82) [reply ui/components/common/Validators.js]: You don't pass the name. You only pass the validation function and the value. I'll change it to exit early when the result is false.

Comment (pb82) [reply ui/components/common/Validators.js]: @camilamacedo86 @josemigallas https://github.com/aerogear/data-sync-ui/pull/70

**Pull request closed without merging**